src/compiler/checker.ts

```diff
@@ -5988,8 +5988,17 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
     }
 
     function symbolToString(symbol: Symbol, enclosingDeclaration?: Node, meaning?: SymbolFlags, flags: SymbolFormatFlags = SymbolFormatFlags.AllowAnyNodeKind, writer?: EmitTextWriter): string {

```

Comment:
I'm moderately worried that defaulting like this will lead us to accidentally using the "wrong" quote style in places where it matters, but maybe I'm overthinking it.

Reply:
`symbolToString` is used in the checker only when building errors (i did check all occurrences in side the checker). The only cause for concern is that it is also used in that it is also used in  `getFullyQualifiedName` which in TS is only used in the display name for renames, but it is part of the public API so some people using it and will see the new default.